ArcGISRuntimeSDKQt_CppSamples/AR/DisplayScenesInTabletopAR/PermissionsHelper.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 ESRI
```

Comment:
This is a new file in this repo, so should be 2020. This applies to all the source/header files.